source/view.c

```diff
@@ -540,25 +553,95 @@ static void rofi_view_call_thread ( gpointer data, gpointer user_data )
     g_mutex_unlock ( t->mutex );
 }
 
+enum CharClass { LOWER, UPPER, DIGIT, NON_WORD };
+
+static enum CharClass char_class(gunichar c)
```

Comment:
I'll change the name to `rofi_sort_get_character_class` because it is not a re-implementation of `fzf`'s fuzzy finding algorithm. (I have read its code and thought it difficult to migrate its code to rofi because it is too complex.)